adapters/wpcom-vip.php

```diff
@@ -46,8 +47,9 @@ protected function set_posts( $q, $es_response ) {
 							$post__in = implode( ',', $post_ids );
 							$this->posts = $wpdb->get_results( "SELECT $wpdb->posts.* FROM $wpdb->posts WHERE ID IN ($post__in) ORDER BY FIELD( {$wpdb->posts}.ID, $post__in )" );
 						}
-						return;
 					}
+
+					$this->posts = $this->post_query_sort_handler( $this->posts, $q );
```

Comment:
For consistency with the rest of the `switch`, I would add a `return;` after this statement.